### ISSUE TITLE
IBGNetworkLog saveRequestWithUrl:body Crashs

### ISSUE THREAD

**Issue by tgensol** [Instabug/Instabug-React-Native#613]
Hi there,

I released a new version of my app last week with the latest version of Instabug-RN, and since that day my app is really unstable : 

![Capture d’écran 2021-08-16 à 18 09 34](https://user-images.githubusercontent.com/7864896/129594865-67fce73a-b7a8-447c-a56d-6d7424abd0c6.png)

I am getting lots of the same function `IBGNetworkLog saveRequestWithUrl:body`

It just changed to a Major issue on my app, I will try to downgrade the previous version to see

Any idea ? Thanks for the help !

**Comment by julianokhalife** [comment]
I had the same issue. It's occurring on iOS only and I am getting these issues whenever the internet connection is not stable.
To reproduce the crash:
- Close the application from the background
- Turn off the Wifi/4G
- Open the application and the crash should occur

**Comment by haozhutw** [comment]
Same issue happens to my app when I upgraded instabug-reactnative to v10.8.0. I want to resolve crashes caused by Instabug but it turns out more crashes are happening, so I have to rollback the upgrade.

I have to say that Instabug has great features, but its stability is really bad! I have seen too many crashes caused by Instabug. As a third party SDK, it's not stable. Think about that, a SDK collects and analyze crash logs but eventually found that most crashes are caused by itself! What a joke!

**Comment by AliAbdelfattah** [comment]
For anyone facing this crash, we've pushed a snapshot that resolves the crash. I'll keep the issue open until it gets released.

```
npm install --save https://github.com/Instabug/Instabug-React-Native#snapshot/10.8-ios-network-body
```

**Comment by AliAbdelfattah** [comment]
Fix released in v10.8.1.
